distribution/Containerfile.in

```diff
@@ -3,9 +3,9 @@ WORKDIR /opt/app-root
 
 RUN pip install sqlalchemy # somehow sqlalchemy[asyncio] is not sufficient
 {dependencies}
-RUN pip install --no-cache llama-stack==0.2.14
+RUN pip install --no-cache llama-stack==0.2.18
 RUN mkdir -p ${{HOME}}/.llama/providers.d ${{HOME}}/.cache
-COPY redhat-distribution/run.yaml ${{APP_ROOT}}/run.yaml
-COPY redhat-distribution/providers.d/ ${{HOME}}/.llama/providers.d/
+COPY distribution/run.yaml ${{APP_ROOT}}/run.yaml
+COPY distribution/providers.d/ ${{HOME}}/.llama/providers.d/
 
-ENTRYPOINT ["python", "-m", "llama_stack.distribution.server.server", "--config", "/opt/app-root/run.yaml"]
+ENTRYPOINT ["python", "-m", "llama_stack.core.server.server", "/opt/app-root/run.yaml"]
```

Comment:
so this is diff for `0.2.17` as before 

today we had this issue raised and @leseb fixed it in the operator 

i am not sure if we need to take care of this here as well

Reply:
So are we saying this is fine or needs to be changed?

Reply:
> So are we saying this is fine or needs to be changed?

What you have done here is correct as is. We need this fix and the fix in the k8s operator ( https://github.com/llamastack/llama-stack-k8s-operator/pull/153 )